### PR TITLE
fix 'Image slider doesn't load after proxing (http://rutube.ru/)' (close #710)

### LINKED ISSUE #710: Image slider doesn't load after proxing (http://rutube.ru/)
Error from health monitor doesn't reproduce (see https://testcafe-hhhm.devexpress.com/history?timestamp=1470078316293&siteUrl=http://rutube.ru/).

See [video](http://screencast.com/t/PytVgH4K5Ee)

### PR DESCRIPTION
**Pull request by miherlosev**
@churkin @LavrovArtem 
- update `acorn` to the latest version - 4.0.2
- write test

### CODE REVIEW

File: src/processing/script/tools/acorn/index.js [hunk 1]
```diff
@@ -23,18 +18,25 @@
 //
 // [dammit]: acorn_loose.js
 // [walk]: util/walk.js
-//
-// version 3.0.4
```
Comment (LavrovArtem) [inline src/processing/script/tools/acorn/index.js]: Update this comment

File: src/processing/script/tools/acorn/index.js [hunk 2]
```diff
@@ -23,18 +18,25 @@
...
+export {defaultOptions} from "./options"
+export {Position, SourceLocation, getLineInfo} from "./locutil"
+export {Node} from "./node"
+export {TokenType, types as tokTypes} from "./tokentype"
+export {TokContext, types as tokContexts} from "./tokencontext"
+export {isIdentifierChar, isIdentifierStart} from "./identifier"
+export {Token} from "./tokenize"
+export {isNewLine, lineBreak, lineBreakG} from "./whitespace"
+
+export const version = "4.0.2"
```
Comment (LavrovArtem) [inline src/processing/script/tools/acorn/index.js]: Remove unnecessary exports

File: src/processing/script/tools/acorn/index.js [hunk 3]
```diff
@@ -46,3 +48,30 @@ import "./tokenize"
...
+
+// This is a terrible kludge to support the existing, pre-ES6
+// interface where the loose parser module retroactively adds exports
+// to this module.
+export let parse_dammit, LooseParser, pluginsLoose
+export function addLooseExports(parse, Parser, plugins) {
+  parse_dammit = parse
+  LooseParser = Parser
+  pluginsLoose = plugins
+}
```
Comment (LavrovArtem) [inline src/processing/script/tools/acorn/index.js]: Remove unnecessary exports

Comment (miherlosev) [inline src/processing/script/tools/acorn/index.js]: I just copy files from acorn sources.
`acorn` will be periodically update. 
I think we should keep `exports` as is.
@churkin

Comment (inikulin) [inline src/processing/script/tools/acorn/index.js]: It's nice to get rid from everything that we don't need to not grow up our client lib size. I guess we can try to bundle acorn with something like https://github.com/rollup/rollup. Rollup analyzes AST and remove methods and code paths that not used.

Comment (miherlosev) [inline src/processing/script/tools/acorn/index.js]: Ok. It is a separate task.

**Comment by churkin** [comment]
lgtm

**Comment by LavrovArtem** [comment]
lgtm

**Pull request merged**